library/src/main/java/io/andref/rx/widgets/ListViewCard.java

```diff
@@ -76,6 +79,20 @@ public ListViewCard(@NonNull Context context, @Nullable AttributeSet attrs, @Att
         initializeViews(context, attrs, defStyleAttr, defStyleRes);
     }
 
+    @Override
+    protected void onAttachedToWindow()
+    {
+        super.onAttachedToWindow();
+        mCompositeSubscription = new CompositeSubscription();
+    }
+
+    @Override
+    protected void onDetachedFromWindow()
+    {
+        mCompositeSubscription.unsubscribe();
+        super.onDetachedFromWindow();
+    }
```

Comment:
Still don't think we have the right answer here. We need to drive/respond to this view in the consuming activity/fragment, and that lifecycle won't necessarily match window state.

If we're going to subscribe in the view, I think `takeUntil(view detaches)` is more appropriate.